hotsos/core/config.py

```diff
@@ -177,7 +178,8 @@ def __setitem__(self, key, item):
                 item = group.opts[key].value_type(item)
                 break
         else:
-            raise Exception(f"config option '{key}' not found in any optgrpup")
+            raise NameError(
+                f"config option '{key}' not found in any optgrpup")
```

Comment:
There is a typo here, 'optgrpup', but it was here before.

Reply:
done